js1/vstup-vystup-objekty/konverze-hodnot.md

```diff
@@ -2,61 +2,80 @@
 
 Úplně na začátku je šikovné všimnout si rozdílu mezi hodnotami jako `12` a `'12'`. Jedna představuje číslo dvanáct, druhá představuje řetězec obsahující znaky 1 a 2. Proto můžeme čekat, že aritmetické operace s těmito hodnotami dopadnou jinak podle toho, zda jde o číslo nebo o řetězec.
 
-```jscon
-> 12 + 5
-17
-> '12' + '5'
-'125'
+Tato operace například do stránky vypíše číslo 17:
+
+```js
+document.body.innerHTML += 12 + 5;
+```
+
+Naopak tato operace do stránky vypíše řetězec `125`:
+
+```js
+document.body.innerHTML += '12' + '5';
+```
+
+JavaScript se nám (občas k naší škodě) snaží udělat život jednodušší a tak za nás v určitých případech provádí automatickou konverzi čísel na řetězce. Sami si vyzkoušejte následující příklady a zkuste dopředu odhadnout, co se objeví na stránce.
+
+```js
+document.body.innerHTML += '12' + 5;
 ```
 
-JavaScript se nám (občas k naší škodě) snaží udělat život jednodušší a tak za nás v určitých případech provádí automatickou konverzi.
+```js
+document.body.innerHTML += 12 + '5';
+```
+
+```js
+document.body.innerHTML += 1 + 2 + 3 + '4';
+```
+
+```js
+document.body.innerHTML += '1' + 2 + 3 + 4;
+```
+
+Konverze fungují i pro jiné operátory, napříkad násobení.
+
+```js
+document.body.innerHTML += 12 * 5;
+```
 
-```jscon
-> '12' + 5
-'125'
-> 12 + '5'
-'125'
-> 1 + 2 + 3 + '4'
-'64'
-> '1' + 2 + 3 + 4
-'1234'
+```js
+document.body.innerHTML += '12' * 5;
```

Comment:
> tak za nás v určitých případech provádí automatickou konverzi čísel na řetězce

Navrhuji ten text nahoře přepsat na obecnější `provádí automatickou konverzi` ~čísel na řetězec~, ať to sedí i na tento případ.